indexdata.go

```diff
@@ -178,13 +186,20 @@ func (d *indexData) calculateStats() {
 		DefaultBranchNewLinesCount: defaultCount,
 		OtherBranchesNewLinesCount: otherCount,
 	}
-	d.repoListEntry = RepoListEntry{
-		Repository:    d.repoMetaData,
+	return RepoListEntry{
+		Repository:    d.repoMetaData[i],
 		IndexMetadata: d.metaData,
 		Stats:         stats,
 	}
 }
 
+func (d *indexData) calculateStats() {
+	d.repoListEntry = make([]RepoListEntry, 0, len(d.repoMetaData))
+	for i := 0; i < len(d.repoMetaData); i++ {
+		d.repoListEntry = append(d.repoListEntry, d.calculateStatsForRepoIndex(i))
+	}
+}
```

Comment:
This needs to be accurate before we do our first merge. Otherwise customers will have incorrect information.